packages/console-extension/src/index.ts

```diff
@@ -391,6 +391,11 @@ async function activateConsole(
   });
   await updateSettings();
 
+  // Apply current settings when a console is created.
+  tracker.widgetAdded.connect((sender, panel) => {
+    updateSettings();
```

Comment:
Thanks! You probably noticed that `updateSettings` cycles through each widget and updates settings for each one. What do you think about changing the code so that when a single widget needs an update, like in this case where we have one widget that was added, we only update that single widget rather than updating every widget?

Reply:
Testing that change now 😄